community/gill-jito-airdrop/anchor/README.md

```diff
@@ -0,0 +1,452 @@
+# Solana Merkle Distributor - Modern Implementation
```

Comment:
I'm wondering if you need 2 README`s, one in the root of the project and one in the `anchor` directory. It might make more sense to combine them. Either that, or explicitly link these 2 README's so they point to each other.